stix_shifter_modules/cybereason/stix_transmission/connector.py

```diff
@@ -116,10 +116,10 @@ async def ping_connection(self):
         try:
             response = await self.api_client.ping_box()
             if response.response.history:                           # If the authentication is invalid, the history
-                if response.response.history[0].status_code == 302:  # will be returned with 302 status code.
+                if response.response.history[0].status == 302:  # will be returned with 302 status code.
                     raise InvalidAuthenticationException
             response_code = response.code
-            response_dict = json.loads(response.read().decode('utf-8'))
+            response_dict = json.loads(response.content)
```

Comment:
Please, change this back to `response.read().decode('utf-8')`.
The rest looks good, thank you!

Reply:
Both provide the same results, but we can directly get the response status from the content without having to decode the data.